### PR TITLE
Feature/four color light tower

### PR DESCRIPTION
**Pull request by Antweeker**
Depends on https://github.com/ed-xj/qm/pull/5. We should merge that first.

Change around the buttons in the dashboard view.

### CODE REVIEW

File: ui/web/tpl/DashboardView.html [hunk 1]
```diff
@@ -15,14 +15,13 @@ <h3 class="page-header">
...
+                            <img src="img/machine-diagram.jpg"
```
Comment (ed-xj) [inline ui/web/tpl/DashboardView.html:24]: i don't think we need to keep this diagram in the dashboard view anymore. it should be replaced with hardware(sensor) communication status(online/offline). remember this?

File: ui/web/tpl/DashboardView.html [hunk 2]
```diff
@@ -15,14 +15,13 @@ <h3 class="page-header">
                         <div style="float:left; padding-left:50px;">
                             <div class="btn-group-vertical">
                                 <button type="button" class="btn btn-lg btn-danger" disabled="disabled">Error</button>
-                                <button type="button" class="btn btn-lg btn-warning" disabled="disabled">Warning</button>
-                                <button type="button" class="btn btn-lg btn-success">Normal</button>
-                                <button type="button" class="btn btn-lg btn-primary" disabled="disabled">Idle</button>
-                                <button type="button" class="btn btn-lg btn-default" disabled="disabled">Default</button>
+                                <button type="button" class="btn btn-lg btn-warning" disabled="disabled">Idle</button>
+                                <button type="button" class="btn btn-lg btn-success">Running</button>
+                                <button type="button" class="btn btn-lg btn-primary" disabled="disabled">Unused</button>
```
Comment (ed-xj) [inline ui/web/tpl/DashboardView.html]: For the blue light, we need to have a button without label. so please remove the `Unused` label here.

**Comment by ed-xj** [comment]
hey man, looking good overall, smooth sailing so far? minor change needed and after that. :shipit:

File: ui/web/tpl/DashboardView.html (hunk 1 above)
Comment (Antweeker) [inline ui/web/tpl/DashboardView.html:24]: ahh yeah, I didn't start on that bullet point.

File: ui/web/tpl/DashboardView.html (hunk 2 above)
Comment (Antweeker) [inline ui/web/tpl/DashboardView.html]: bootstrap button acting strangely with a button without text. It resizes it. have to investigate further.

**Comment by Antweeker** [comment]
@ed-xj alright, I removed "Unused" and added an invisible character, good to go?

**Comment by ed-xj** [comment]
let's do it. :shipit:

**Pull request merged**